components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/services/ConditionalAuthenticationMgtService.java

```diff
@@ -44,7 +44,7 @@ public class ConditionalAuthenticationMgtService extends AbstractAdmin {
      */
     public String[] getAllAvailableFunctions() {
 
-        List<String> jsFunctions = Stream.of("executeStep", "selectAcrFrom", "sendError", "Log.info", "require").collect
+        List<String> jsFunctions = Stream.of("executeStep", "selectAcrFrom", "sendError", "Log.info", "require", "secrets.getSecret").collect
                 (Collectors.toList());
```

Comment:
what's the difference from referring functions here and loading from "JsFunctionRegistry"